src/pytest_servers/local.py

```diff
@@ -1,14 +1,7 @@
 import pathlib
 
-from dvc_objects.fs.implementations.local import LocalFileSystem
+from fsspec.implementations.local import LocalFileSystem
 
 
 class LocalPath(type(pathlib.Path())):  # type: ignore[misc]
```

Comment:
pathlib does not have `_accessor ` in 3.11. I thought I tried this way before and it didn't work, but this time it worked idk how. See https://github.com/iterative/dvc-data/pull/75. :)